docs/doc_usage_cmd.md

```diff
@@ -21,7 +21,32 @@ To run DomainLab, the minimum necessary parameters are:
 ### Advanced Configuration
 
 - **Learning Rate (`--lr`):** Set the training learning rate.
-- **Regularization (`--gamma_reg`):** Weight of regularization loss.
+- **Regularization (`--gamma_reg`):** Sets the weight of the regularization
+ loss. This parameter can be configured either as
+ a single value applied to individual classes,
+ or using a dictionary to specify different
+ weights for different models and trainers.
+
+  - **Command Line Usage:**
+    - For a single value: `python script.py --gamma_reg=0.1`
+    - For multiple values: `python script.py --gamma_reg='default=0.1,dann=0.05,diva=0.2'`
```

Comment:
actually, due to the specific implementation of DIVA, instead of $$\ell(\cdot) + \mu R(\cdot)$$ type of loss, for DIVA, it is $$\mu \ell(\cdot) + R(\cdot)$$, here $\mu$ is gamma_reg in the code

Reply:
But where is this specified for model diva. I can only find this initialization of diva which does not have a regularization : 
 model = mk_diva(list_str_y=task.list_str_y)(
            node,
            zd_dim=args.zd_dim,
            zy_dim=args.zy_dim,
            zx_dim=args.zx_dim,
            list_d_tr=task.list_domain_tr,
            gamma_d=args.gamma_d,
            gamma_y=args.gamma_y,
            beta_x=args.beta_x,
            beta_y=args.beta_y,
            beta_d=args.beta_d,
        )
It's [here](https://github.com/marrlab/DomainLab/blob/5ff5fd3d14d20c47fcc741aa0b988dc624d19e67/domainlab/algos/builder_diva.py#L38)

Reply:
it is specified here: https://github.com/marrlab/DomainLab/blob/5ff5fd3d14d20c47fcc741aa0b988dc624d19e67/domainlab/models/model_diva.py#L152

and https://github.com/marrlab/DomainLab/blob/faf7589abf34e2901678b9b6eabfa585ab237851/domainlab/models/model_vae_xyd_classif.py#L31